server.js

```diff
@@ -40,8 +40,8 @@ app.use(helmet.contentSecurityPolicy({
     defaultSrc: ["'none'"],
     fontSrc: ["https://code.cdn.mozilla.net/fonts/"],
     frameAncestors: ["'none'"],
-    imgSrc: ["'self'"],
-    scriptSrc: ["'self'"],
+    imgSrc: ["'self'", "https://www.google-analytics.com", "www.google-analytics.com"],
```

Comment:
Do we need both `https://www...` and `www...`? I'd rather remove the protocol-relative `www...` so our CSP enforces that everything is fetched over `https`

Reply:
Doesn't look like it! Removing..;.